governance/scripts/bridge/status.js

```diff
@@ -18,7 +18,7 @@ const {
 const fs = require('fs-extra')
 
 // small cache to prevent numerous calls to subgraph api
-const filepath = './xcalled.json.tmp'
+const filepath = './xcalled.tmp.json'
```

Comment:
rename that tmp file to `.json` extension so it is parsed and readable in IDE

Reply:
Maybe put this file in `.gitignore`?